src/utils/ethereumMock.ts

```diff
@@ -0,0 +1,69 @@
+import { ethers, Wallet } from 'ethers';
```

Comment:
1. I understand that you forked a project and adapted it to our repo.
Please include an explanation and source links in a comment to share the research. 
2. This is only used in tests. Please move out of `src/utils`. We can place it in `__tests__`?

Reply:
Jujst to clarify, there was no fork done for this PR. It was for the other experiment I did with Web3 E2E, which was dropped. Will move the file.